server/parameters/config/BUILD

```diff
@@ -0,0 +1,28 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+
+filegroup(
+    name = "config_file",
+    srcs = select({
+        "//server:release_mode": ["config_release.yml"],
+        "//server:development_mode": ["config_development.yml"],
+    }),
+    visibility = ["//visibility:public"],
+)
+
+genrule(
+    name = "config",
+    srcs = [":config_file"],
+    outs = ["config.yml"],
+    cmd = "cp $(SRCS) $@",
+    visibility = ["//visibility:public"],
```

Comment:
you'll need to make a `cmd_bat` addition here so that this can run on windows I think

Reply:
Good catch, I didn't notice that we don't use already existing similar genrules for Windows.